alacritty/src/display/mod.rs

```diff
@@ -425,15 +437,15 @@ impl Display {
         // If the scaling factor changed update the glyph cache and mark for resize.
         let should_resize = (estimated_scale_factor - window.scale_factor).abs() > f64::EPSILON;
         let (cell_width, cell_height) = if should_resize {
-            Self::update_glyph_cache(&mut renderer, &mut glyph_cache, scale_factor, config, font)
+            Self::update_font_size(&mut glyph_cache, scale_factor, config, font)
         } else {
             (cell_width, cell_height)
         };
 
         // Load font common glyphs to accelerate rendering.
         debug!("Filling glyph cache with common glyphs");
         renderer.with_loader(|mut api| {
-            glyph_cache.load_common_glyphs(&mut api);
+            glyph_cache.clear_glyph_cache(&mut api);
```

Comment:
I feel like the `clear_glyph_cache` function has a pretty bad name. I actually had to look it up just to make sure it actually loads the common glyphs.

Maybe something like `reset_glpyh_cache` would be more appropriate? Just to indicate it is reset to a specific state, not emptied?